src/index.js

```diff
@@ -1,47 +1,174 @@
 import './style.css';
-
 // input section
 import Image from './asset/images/add-grey.svg';
 import Image1 from './asset/images/3dots-grey.svg';
+import Image2 from './asset/images/trash.svg';
 
 const render = `<a href="#"><img src="${Image}" alt="img"></a>`;
 const addButton = document.getElementById('add-button');
 addButton.innerHTML = render;
 
 // list section
-const ListContent = [
-  {
-    id: 0,
-    description: 'task1',
-    status: false,
-  },
-  {
-    id: 1,
-    description: 'task2',
-    status: false,
-  },
-  {
-    id: 2,
-    description: 'task3',
-    status: false,
-  },
-  {
-    id: 3,
-    description: 'task4',
-    status: false,
-  },
-];
-
-// iteration through array
-ListContent.forEach((todo) => {
+let ListContent = [];
+
+// functions
+// status checkbox
+
+const statusCheckBox = (e) => {
+  const list = e.target.parentNode.parentNode;
+  list.children[0].children[1].classList.toggle('strike');
+  const completed = !list.classList.contains('disabled');
+  list.classList.toggle('disabled');
+  const getLists = document.querySelector('#ulList').children;
+  const index = Array.from(getLists).indexOf(list);
+  ListContent[index].completed = completed;
+  localStorage.setItem('tasks', JSON.stringify(ListContent));
+};
+
+// sorting index values
+const sortIndex = () => {
+  ListContent = ListContent.map((item, index) => ({
+    index: index + 1,
+    description: item.description,
+    completed: item.completed,
+  }));
+  localStorage.setItem('tasks', JSON.stringify(ListContent));
+};
+
+// update local storage
+const updateLocalStorage = (arr, item) => {
+  const index = Array.from(arr).indexOf(item.parentNode.parentNode);
+  ListContent[index].description = item.value;
+  localStorage.setItem('tasks', JSON.stringify(ListContent));
+};
+
+// input validation
+const isValid = (value) => {
+  if (!value) {
+    return false;
+  }
+  return true;
+};
+
+// after editing todo list - hit enter to save
+const saveValue = (event) => {
+  if (
+    event.target.classList.contains('todo-description')
+    && event.keyCode === 13
+  ) {
+    event.target.disabled = true;
+    event.target.parentNode.parentNode.children[1].src = Image1;
+    const todoContent = document.querySelector('#ulList').children;
+    updateLocalStorage(todoContent, event.target);
+    sortIndex();
+  }
+};
+
+// enable editing and removing todolist
+const todoEdit = (event) => {
+  if (event.target.src === Image1) {
+    event.target.src = Image2;
+    event.target.parentNode.children[0].children[1].removeAttribute('disabled');
+    // document.querySelector(".todo-description").removeAttribute("disabled");
+  } else {
```

Comment:
- [ ] Nice job working on the requirement from the other code reviewer. You are close to approval for this milestone. Kindly remove all codes that are commented out in your project. Removing such codes helps keep your code clean. 👍

Reply:
Sure @Gambit142. I'll do it now. Thanks for helping me to improvise and learn. :+1: